diff --unified --recursive --text --color appgate-sdp.orig/opt/appgate/linux/nm.py appgate-sdp.new/opt/appgate/linux/nm.py
--- appgate-sdp.orig/opt/appgate/linux/nm.py	2020-11-23 09:10:54.000000000 +0000
+++ appgate-sdp.new/opt/appgate/linux/nm.py	2021-01-14 18:45:00.930430533 +0000
@@ -276,7 +276,9 @@
         syslog(LOG_INFO, '[has_system_dnsmasq] dnsmasq bus present')
         return True
 
-    if distro.linux_distribution()[0] != 'Fedora':
+    # this is PKGBUILD for Arch so this check should always yield True
+    # if platform.linux_distribution()[0] != 'Fedora':
+    if True:
         try:
             bus = dbus.SystemBus()
             proxy = bus.get_object(
